src/test/daml/Daml/Finance/Test/Util/Holding.daml

```diff
@@ -36,3 +39,24 @@ verifyNoObservers l = forA l
       exerciseCmd (toInterfaceContractId @Disclosure.I holdingCid)
         Disclosure.GetView with viewer = owner
     v.observers === empty
+
+-- | Create a holding factory with refernce.
+createHoldingFactoryWithReference :
```

Comment:
I think we can just call it `createHoldingFactory`

Reply:
Agree